docs/ispc.rst

```diff
@@ -1018,6 +1018,21 @@ Neon targets:
 
 ``neon-i8x16``, ``neon-i16x8``, ``neon-i32x4``, ``neon-i32x8``.
 
+The following table lists the ISPC targets and their corresponding architecture details for ARM.
+
+============================= ============ ==============
+ISPC target/arch              ARM arch        ARM ISA
```

Comment:
I can combine 2 columns into one like:
ARMv8-A AARCH32 -> ARMv8-A 32-bit
ARMv8-A AARCH64 -> ARMv8-A 64-bit

Reply:
Ah, ok, so your point is to highlight that it's 32 bit ARMv8 vs 64 bit ARMv8. And not an old ARMv7. I see. Combined single column is more clear, I think.

But again, I think we should enable the whole matrix eventually. Pros: it should be easy because of generic target, it's a clear mental model for the user. Cons: it requires testing.

Reply:
yeah, here the table just shows the current state of things (which was not clear for me at all at the beginning)

Reply:
updated